samples/MyEf.Hr/README.md

```diff
@@ -59,63 +59,23 @@ Endpoint | Description
 `DELETE /reviews/id` | Delete an existing review.
 `GET /employee/id/reviews` | Gets all review(s) for the employee (with paging support).
 
-</br>
-
-## Solution skeleton
-
-This solution should be created using the solution [template](../../templates/Beef.Template.Solution/README.md) capability, following the getting started [guide](../../docs/Sample-SqlServer-EF-GettingStarted.md).
-
-The following four commands should be invoked to create the solution structure. Start in a folder where the solution should reside. To simplify the ongoing copy and paste activities within this sample it is _highly recommended_ that the `MyEf.Hr` naming convention below is used.
-
-```
-dotnet new install beef.template.solution --nuget-source https://api.nuget.org/v3/index.json
-mkdir MyEf.Hr
-cd MyEf.Hr
-dotnet new beef --company MyEf --appname Hr --datasource SqlServer
-```
-
-The following solution structure will have been generated. Open `MyEf.Hr.sln` in Visual Studio.
-
-```
-└── MyEf.Hr               # Solution that references all underlying projects
-  └── Testing
-    └── MyEf.Hr.Test      # Unit and intra-integration tests
-  └── Tools
-    └── MyEf.Hr.CodeGen   # Entity and Reference Data code generation console
-    └── MyEf.Hr.Database  # Database code generation console
-  └── MyEf.Hr.Api         # API end-point and operations
-  └── MyEf.Hr.Business    # Core business logic components
-  └── MyEf.Hr.Common      # Common / shared components
-```
-
-_Note:_ Code generation should **not** be performed before updating the corresponding YAML files as described in the next sections. Otherwise, extraneous files will be generated that will then need to be manually removed.
-
-Also, any files that start with `Person` (being the demonstration entity) should be removed (deleted) from their respective projects as they are encountered. This then represents the baseline to build up the solution from.
-
-</br>
-
-## Railway-oriented programming
```

Comment:
This section is removed; it need to be moved somewhere as it is important?

Reply:
Most moved into separate `1-Solution-Skeleton.md` page.  readme now also includes the railway section.  I must have accidentally removed it.